src/components/organisms/tasks/twitter-connection-card.tsx

```diff
@@ -1,6 +1,6 @@
 import useTranslation from 'next-translate/useTranslation';
 
-import { ConnectionHandlerTwitter } from '@/components/templates/settings/connected-accounts/connectors/twitter-connection';
+import { ConnectionHandlerTwitter } from '@/services/social-connectors/twitter-connection';
```

Comment:
please, prefix both handlers with `use` (on the function itself at `@/services/social-connectors/x`). 
I've checked both methods and they're are hooks, so we must tell to react that. (and hooks starts with `use`)

```suggestion
import { useConnectionHandlerTwitter } from '@/services/social-connectors/twitter-connection';
```